test/unit/overlay.js

```diff
@@ -6,19 +6,78 @@ var sharp = require('../../index');
 
 sharp.cache(0);
 
-// Main
+
+// Constants
+var MAX_ALLOWED_IMAGE_MAGICK_MEAN_SQUARED_ERROR = 0.3;
```

Comment:
:arrow_down: Turns out `sharp`’s results are really close to ImageMagick, as one would expect :wink: